recipe/bld.bat

```diff
@@ -24,3 +24,5 @@ copy prog\udunits2.exe %SCRIPTS%\udunits2.exe || exit 1
 copy lib\udunits2.dll %SCRIPTS%\udunits2.dll || exit 1
 copy lib\*.exp %LIBRARY_LIB%\ || exit 1
 copy lib\*.lib %LIBRARY_LIB%\ || exit 1
+
+dir %LIBRARY_PREFIX%\share\udunits\
```

Comment:
Thanks @msarahan! I will check the source code to see what we can do. If not I will downgrade to conda-build `2.1.1` temporarily here just to get this merged.